content/en/docs/tasks/administer-cluster/running-cloud-controller.md

```diff
@@ -77,18 +74,19 @@ Cloud controller manager does not implement any of the volume controllers found
 
 ### Scalability
 
-In the previous architecture for cloud providers, we relied on kubelets using a local metadata service to retrieve node information about itself. With this new architecture, we now fully rely on the cloud controller managers to retrieve information for all nodes. For very large clusters, you should consider possible bottle necks such as resource requirements and API rate limiting.
+The cloud controller queries your cloud provider's APIs to retrieve information for all nodes. For very large clusters, you should consider possible bottle neckssuch as resource requirements and API rate limiting.
 
 ### Chicken and Egg
 
 The goal of the cloud controller manager project is to decouple development of cloud features from the core Kubernetes project. Unfortunately, many aspects of the Kubernetes project has assumptions that cloud provider features are tightly integrated into the project. As a result, adopting this new architecture can create several situations where a request is being made for information from a cloud provider, but the cloud controller manager may not be able to return that information without the original request being complete.
 
-A good example of this is the TLS bootstrapping feature in the Kubelet. Currently, TLS bootstrapping assumes that the Kubelet has the ability to ask the cloud provider (or a local metadata service) for all its address types (private, public, etc) but cloud controller manager cannot set a node's address types without being initialized in the first place which requires that the kubelet has TLS certificates to communicate with the apiserver.
+A good example of this is the TLS bootstrapping feature in the Kubelet. TLS bootstrapping assumes that the Kubelet has the ability to ask the cloud provider (or a local metadata service) for all its address types (private, public, etc) but cloud controller manager cannot set a node's address types without being initialized in the first place which requires that the kubelet has TLS certificates to communicate with the apiserver.
```

Comment:
nit: Is the `Chicken and Egg` heading easily translated?

Reply:
How about fixing that in a different PR? The “Chicken and Egg” text is not a regression.

Reply:
👍 This is not as important as sorting out some of the proposed changes.